components/scoreHandler.ts

```diff
@@ -858,6 +847,17 @@ export async function missionEnd(
     let unlockableProgression = undefined
     let sniperChallengeScore = undefined
 
+    let contractScore = {
```

Comment:
I guess this could work, since the sniper game-mode doesn't use it. Please squash when merging so it's only this change :)